awx/main/analytics/subsystem_metrics.py

```diff
@@ -166,7 +166,11 @@ def __init__(self, auto_pipe_execute=False, instance_name=None):
         elif settings.IS_TESTING():
             self.instance_name = "awx_testing"
         else:
-            self.instance_name = Instance.objects.me().hostname
+            try:
+                self.instance_name = Instance.objects.me().hostname
+            except Exception as e:
+                self.instance_name = settings.CLUSTER_HOST_ID
+                logger.info(f'Instance {self.instance_name} seems to be unregistered, error: {e}')
```

Comment:
@fosterseth help me out here please. This subsystem metrics code is front-running a million other scenarios by throwing an exception before we get deeper into the service details. For an example, we have this log which is beautifully constructed:

```
tools_awx_1 | 2022-07-08 17:40:57,095 INFO     [-] awx.main.wsbroadcast Unable to return currently active instance: No instance found with the current cluster host id, retry in 5s...
tools_awx_1 | make[1]: Leaving directory '/awx_devel'
tools_awx_1 | 2022-07-08 17:41:02,473 INFO exited: awx-wsbroadcast (exit status 0; expected)
tools_awx_1 | 2022-07-08 17:41:03,477 INFO spawned: 'awx-wsbroadcast' with pid 27242
tools_awx_1 | make[1]: Entering directory '/awx_devel'
tools_awx_1 | 2022-07-08 17:41:04,498 INFO success: awx-wsbroadcast entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
tools_awx_1 | 2022-07-08 17:41:05,510 INFO     [-] awx.main.wsbroadcast Unable to return currently active instance: No instance found with the current cluster host id, retry in 5s...
tools_awx_1 | make[1]: Leaving directory '/awx_devel'
tools_awx_1 | 2022-07-08 17:41:10,892 INFO exited: awx-wsbroadcast (exit status 0; expected)
tools_awx_1 | 2022-07-08 17:41:10,894 INFO spawned: 'awx-wsbroadcast' with pid 27275
tools_awx_1 | make[1]: Entering directory '/awx_devel'
tools_awx_1 | 2022-07-08 17:41:11,903 INFO success: awx-wsbroadcast entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
tools_awx_1 | 2022-07-08 17:41:12,996 INFO     [-] awx.main.wsbroadcast Unable to return currently active instance: No instance found with the current cluster host id, retry in 5s...
```

I can get this if I suppress errors from the code here, as I'm trying to do here.

But this begs the obvious question, why do this at all? Why did we not start out referencing `settings.CLUSTER_HOST_ID`? We don't need the _model_, just the _name_. Did we get ourselves into this situation by cargo-culting the `Instance.objects.me()` call?